apps/trading/lib/utils/is-asset-usdt-arb.ts

```diff
@@ -0,0 +1,14 @@
+import type { AssetFieldsFragment } from '@vegaprotocol/assets';
+import {
+  ARBITRUM_CHAIN_ID,
+  ARBITRUM_SEPOLIA_CHAIN_ID,
+} from '@vegaprotocol/web3';
+
+export const isAssetUSDTArb = (asset: AssetFieldsFragment) => {
+  const source = asset.source.__typename === 'ERC20' ? asset.source : undefined;
+  return (
+    asset.symbol.toLowerCase() === 'usdt' &&
```

Comment:
We've run into clashes of symbol before. Though its very unlikely on mainnet, I'd prefer if we just had the arb usd asset ID hardcoded in a config file and we used that, we'd then only need to check the id